cmd/apps/kubernetes_dashboard_app.go

```diff
@@ -72,14 +102,19 @@ subjects:
 ---
 EOF
 
-#To forward the dashboard to your local machine 
-kubectl proxy
+# To forward the dashboard to your local machine 
+kubectl -n kubernetes-dashboard port-forward services/kubernetes-dashboard 8443:443
+
+# To get your Token for logging in
+
+## K8s v1.24 or above
+kubectl -n kubernetes-dashboard create token admin-user
```

Comment:
What do they do after running `kubectl -n kubernetes-dashboard create token admin-user`?

What if they already created that and are running `arkade info`?

Reply:
It's just basically a better way to get a token.
Should we use it with the `--bound-object-kind Secret` and `--bound-object-name secretName`, to persist the generated token or is it fine to generate a new token every time?

Reply:
I don't know

Could you change it to show both commands instead?

Reply:
Both commands now present in the info message.